docs/reference/settings/ilm-settings.asciidoc

```diff
@@ -1,8 +1,12 @@
 [role="xpack"]
 [[ilm-settings]]
-=== {ilm-cap} settings
+=== {ilm-cap} settings in {es}
```

Comment:
I'm not sure `in {es}` is adding much here, but it's not a big deal. I would assume users know they're in the ES docs.

Reply:
I was following the pattern in some of the other settings topics. I don't feel strongly one way or the other. It's possibly helpful from an SEO perspective, but with elasticsearch in the URL, it's likely negligible.  I'll open a follow on to make them all consistent.